pkg/sink/codec/common/message.go

```diff
@@ -172,30 +170,15 @@ func UnmarshalClaimCheckMessage(data []byte) (*ClaimCheckMessage, error) {
 	return &m, err
 }
 
-// NewClaimCheckFileName return file name for sent the message to claim check storage.
-// make sure the file name can identify one event uniquely.
-// {date}/{schema}-{table}-{commitTs}-{startTs}-{handleKeys}.json
-// the files is organized by date, it's easy to clean up those objects by removing the whole directory.
-func NewClaimCheckFileName(e *model.RowChangedEvent) string {
```

Comment:
Perhaps this logic can be retained to meet the possible needs of users in future?